rplugin/python3/deoplete/sources/swift.py

```diff
@@ -62,19 +62,16 @@ def gather_candidates(self, context):
     def identifiers_from_result(self, result):
         out = []
 
-        candidates = []
         longest_desc_length = 0
         longest_desc = ''
         for complete in result:
-            candidates.append(complete)
-
             desc_len = len(complete['name'])
 
             if desc_len > longest_desc_length:
                 longest_desc = complete['name']
                 longest_desc_length = desc_len
 
-        for completion in candidates:
+        for completion in result:
```

Comment:
Thanks. This was a result of using deoplete-d as a starting point for this project iirc